### DISCUSSION TITLE
CS%20Meng%40Cornell

### DISCUSSION THREAD

**Discussion by xichenpan** [opencsapp/opencsapp.github.io#72] [Announcements]
# CS%20Meng%40Cornell/

https://csmsapp.github.io/CS%20Meng%40Cornell/

**Comment by wendy556** [comment]
【21fall】在网上查其他项目的时候看到了一个案例（https://zhuanlan.zhihu.com/p/387177147）
学校及专业：北京大学 计算机  硬件成绩：GPA:3.6 TOEFL:108 GRE:328
贴上来学员访谈：https://mp.weixin.qq.com/s?__biz=MzA5NjUwMTgxOQ==&mid=2652155296&idx=1&sn=467a688aa5e5efb2e24ccf8a1a51d906&chksm=8b4f24f4bc38ade23d69671a03bacdcf111e8a837bd5467fcb54573956ecd291d3491c523417&scene=21#wechat_redirect

**Reply by wendy556** [reply]
不好意思这个是Cornell Tech CS Meng，我写错地方了（

**Comment by CindyCindy424** [comment]
想问一下什么叫网申支持interfolio呀

**Comment by b03505036** [comment]
贡献Cornell CS MEng个dp

本科：UIUC CS GPA: 3.97/4.0
三段实习，两段美国大厂SDE, 一段中厂

**Comment by Snackiller** [comment]
现在延不了了

**Reply by EthanJiang02** [reply]
你好现在延不了了嘛

**Comment by wocqcm2** [comment]
+一个25fall Cornell CS meng ithaca dp. 
本科 OSU CSE 3.96/4
本校 一段水科研 一段水竞赛
+一段超级水实习
GRE 324 + 3 不过好像没看

**Comment by wendy556** [comment]
延什么？

---- Replied Message ----
| From | ***@***.***> |
| Date | 03/22/2026 03:03 |
| To | ***@***.***> |
| Cc | wendy556 ***@***.***>,
Comment ***@***.***> |
| Subject | Re: [opencsapp/opencsapp.github.io] CS%20Meng%40Cornell (Discussion #72) |

你好现在延不了了嘛
